packaging_automation/citus_package.py

```diff
@@ -313,6 +315,15 @@ def get_package_version_from_pkgvars(input_files_dir: str):
     return package_version
 
 
+def get_package_version_without_release_stage_from_pkgvars(input_files_dir: str):
```

Comment:
What does release stage mean here ? Or, can you give an example about what does this function return for an example input, for me to better understand ?

Reply:
what about we call it as release_type then ?